lib/also_energy/client.rb

```diff
@@ -21,7 +26,7 @@ def initialize()
     def login
       message = { 'als:username' => username, 'als:password' => password }
       response = find_in_hash(:session_id, (connection.call(:login, message: message).body))
-      response == nil ? (fail AuthError, "Login Failed!") : (self.session_id = response)
+      response == nil ? (fail AuthError, "Login Failed!") : (self.session_id = response && AlsoEnergy.session_id = response)
```

Comment:
This line is a bit unruly. I would opt for a standard if then block in this case.